config/initializers/sentry.rb

```diff
@@ -0,0 +1,17 @@
+require 'sentry-ruby'
+
+# in Rails, this might be in config/initializers/sentry.rb
+Sentry.init do |config|
+  config.dsn = ENV['SENTRY_DSN']
+  config.environment = ENV['SENTRY_ENVIRONMENT']
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+  # Set traces_sample_rate to 1.0 to capture 100%
+  # of transactions for performance monitoring.
+  # We recommend adjusting this value in production.
+  config.traces_sample_rate = 1.0
```

Comment:
this should be 0.1 for prod please

Reply:
hard coded or as an environment variable?